base/CMakeLists.txt

```diff
@@ -46,12 +54,16 @@ therock_cmake_subproject_glob_c_sources(rocm_smi_lib
 therock_cmake_subproject_provide_package(rocm_smi_lib rocm_smi lib/cmake/rocm_smi)
 therock_cmake_subproject_activate(rocm_smi_lib)
 
+endif()
+
 
 ################################################################################
 # rocprofiler-register
 # This is a stub that helps runtime libraries and profiles rendezvous
 ################################################################################
 
+if(NOT WIN32)  # TODO(#36): Enable on Windows and/or make subproject inclusion generally optional
+
 therock_cmake_subproject_declare(rocprofiler-register
   EXTERNAL_SOURCE_DIR "rocprofiler-register"
```

Comment:
Argh, and this also won't work due to a dep here: https://github.com/nod-ai/TheRock/blob/b3307edafb25943df17c0bdb022548ab65e7338e/core/CMakeLists.txt#L5-L13

I had this "working" on Friday with my build directory, but now seeing issues with a fresh build.

Reply:
Yeah... That's a non optional dep to a few things

Reply:
Okay, I'm going in circles here. This is actually fine, as long as I have 'core' also excluded, which I do:
```
if(NOT WIN32)
  add_subdirectory(compiler)
  add_subdirectory(core)
```

Reply:
Picking this back up with a fresh set of eyes and some hacks to get a CI runner past the symlink issues.

The excludes that I have here are sufficient to build, but they aren't a comfortable solution. I'd like to proceed with this approach for now then see about making rocprofiler-register (and other projects) compile successfully but no-op in some/all of its functionality as needed.